src/pages/task/FilesTables.tsx

```diff
@@ -117,6 +117,7 @@ export const FilesTables: React.FC = () => {
         onChange={onSearch}
         suffix={<Icon glyph="MagnifyingGlass" />}
       />
+      <br />
```

Comment:
Why the br. Can we just do the padding with a margin on the styled input?

Reply:
the br is actually so the input is on a separate line from the contents below. an alternative would be to wrap the input with a div

Reply:
I like wrapping the parent with a flex column div approach a little better. The br seems a little out of place, In this context.
Typically `<br/>`  is [used to break up lines of  text instead of elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#:~:text=The%20HTML%20element%20produces,division%20of%20lines%20is%20significant.)